app/Notifications/JoinGroup.php

```diff
@@ -74,19 +83,23 @@ public function toMail($notifiable)
         } else { // users not yet on the platform
```

Comment:
This one is an interesting one - it is being sent to users not currently on the platform.  So not sure what `notifiable->language` will be.  We should probably create a separate ticket for this specific notification to think about how to deal with this (and the one for inviting to events, where presumably the same could happen).  We could maybe take the locale from the network the group is in (but then, what if it's in two networks etc etc...)

Reply:
Created https://therestartproject.atlassian.net/browse/RES-1625